content/rc/rc-quickstart.md

```diff
@@ -26,20 +26,31 @@ If you already have a subscription, see [Manage subscriptions]({{< relref "/rc/s
 
 To create a new account with a free subscription and database:
 
-1. On the Redis Cloud [admin console](https://app.redislabs.com/#/login), select [Sign up](https://redis.com/try-free/).
+1. Navigate to the [Sign up](https://redis.com/try-free/) page.
 
-1. Enter your information in the form and select **Get Started**, or sign up with Google or Github.
+1. There are two options available to get started with Redis Cloud:
+   * Enter your information in the form and select **Get Started**
+   * Sign up with **Google** or **Github**
 
-1. In the activation email, select **Activate account** to go to the Redis Cloud [admin console](https://app.redislabs.com/#/login).
+1. Once you have completed step 2, you should receive an activation email from Redis. In the activation email, select **Activate account**. You will be forwarded to the **Get Started** page.
 
-    {{<image filename="images/rc/quickstart-quick-dialog.png" width="75%" alt="Dialog to create your free subscription." >}}{{< /image >}}
+    {{<image filename="images/rc/quickstart-get-started.png" width="75%" alt="Dialog to create your free subscription." >}}{{< /image >}}
+
+   * Choose your role
+   * Choose who you work for
+   * Choose your use case
+   * Choose your client
+
+    Click **Create Free Database**. You also have the option to **Create a custom database** if necessary.
```

Comment:
```suggestion
    Click **Create Free Database** to move on.
```
We talk about choosing **create a custom database** in the note two steps down. I don't think we need to talk about creating a custom database here.